cmd/beacon-chain/jwt/jwt.go

```diff
@@ -0,0 +1,69 @@
+package jwt
+
+import (
+	"encoding/hex"
+	"math/rand"
+	"os"
+	"time"
+
+	"github.com/prysmaticlabs/prysm/cmd"
+	"github.com/prysmaticlabs/prysm/runtime/tos"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+var log = logrus.WithField("prefix", "jwt")
+
+// Commands for interacting with jwt tokens that support the beacon node.
+var Commands = &cli.Command{
+	Name:     "jwt",
+	Category: "jwt",
+	Usage:    "defines commands for interacting with jwt tokens that support the beacon node",
+	Subcommands: []*cli.Command{
+		{
+			Name:        "generate-jwt-secret",
+			Description: `creates a random 32 byte hex string in a plaintext file within the root /prysm directory`,
+			Flags:       cmd.WrapFlags([]cli.Flag{}),
+			Before:      tos.VerifyTosAcceptedOrPrompt,
+			Action: func(cliCtx *cli.Context) error {
+				if err := generateHttpSecretInFile(); err != nil {
```

Comment:
Before calling this func, you can get the value of the output-file flag by doing something like:
```
outputFile := cliCtx.String(flags.OutputFileFlag.Name)
```
and pass it into the generateHttpSecretInFile func